src/drive/web/modules/upload/index.js

```diff
@@ -171,12 +234,25 @@ const getErrors = queue => filterByStatus(queue, FAILED)
 const getQuotaErrors = queue => filterByStatus(queue, QUOTA)
 const getNetworkErrors = queue => filterByStatus(queue, NETWORK)
 const getLoaded = queue => filterByStatus(queue, LOADED)
```

Comment:
If we remove `loaded` we should remove the getter too?

Reply:
the `getLoaded` is used in `getUploaded` and `getUpdated`